docs/api/logger.rst

```diff
@@ -1,6 +1,23 @@
 Logger module
 =============
 
+Each module of :program:`rebase-helper` uses a dedicated logger. Its name corresponds to
+the position in the module hierarchy (e.g. module :mod:`rebasehelper.application` uses the logger `rebasehelper.application`).
+The :class:`rebasehelper.logger.CustomLogger` class is used instead of `logging.Logger` to add more levels of logging.
+
+By default, all loggers are set to logging level `DEBUG` to ensure that all messages are caught.
```

Comment:
This is unimportant. End users don't care, and they get log files of 3 verbosity levels anyway.